src/SGB_Command_Undocumented.md

```diff
@@ -1,14 +1,14 @@
 # Undocumented SGB commands
 
-The following information has been extracted from disassembling a SGB1v2
+The following information has been extracted from disassembling an SGB1v2
```

Comment:
I don't think SGB is pronounced as an acronym here, but as Super Game Boy

Reply:
That's right, thanks for the review. I pushed a commit undoing the change.